### PR TITLE
reset.asc: Mention "working tree" as equivalent to "working directory"

### PR DESCRIPTION
**Pull request by rpjday**
Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>

### CODE REVIEW

File: book/07-git-tools/sections/reset.asc [hunk 1]
```diff
@@ -71,7 +71,7 @@ The index is not technically a tree structure -- it's actually implemented as a
 
 ===== The Working Directory
 
-Finally, you have your working directory.
+Finally, you have your working directory (also referred to in much of the literature as the ``working tree'').
```
Comment (ben) [inline book/07-git-tools/sections/reset.asc]: Seems a bit heavy. Maybe `sometimes referred to as the ``working tree''`?

**Comment by rpjday** [comment]
Will do, shortly.

**Comment by zoidhddhdd** [comment]
cool

File: book/07-git-tools/sections/reset.asc (hunk 1 above)
Comment (ben) [reply book/07-git-tools/sections/reset.asc]: In Asciidoc, you have to opt in to curly quotes, and their direction matters. Check [the cheatsheet](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/) and scroll down to "Curved Quotation Marks".

**Pull request closed without merging**